Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -142,7 +168,7 @@ void UPSWorldSubsystem::Deinitialize()
 	Super::Deinitialize();
 	PSHUDComponentInternal = nullptr;
 	PSCurrentSpotComponentInternal = nullptr;
-	SaveGameInstanceInternal = nullptr;
+	SaveGameDataInternal = nullptr;
 	SpawnedStarActorsInternal.Empty();
 	StarDynamicProgressMaterial = nullptr;
```

Comment:
Please do `ProgressionSettingsDataInternal.Empty()` of new array here

Reply:
done: https://github.com/h4rdmol/ProgressionSystem/pull/18/commits/de71a966be19781a5caa410920d75e6fcefe4a73